velox/functions/prestosql/aggregates/ApproxDistinctAggregate.cpp

```diff
@@ -111,7 +112,38 @@ struct HllAccumulator {
 
 template <typename T>
 inline uint64_t hashOne(T value) {
-  return XXH64(&value, sizeof(T), 0);
+  if constexpr (std::is_integral_v<T> && !std::is_same_v<T, int64_t>) {
```

Comment:
Can you check we are not narrowing `int128_t` down here?

Reply:
Removed the narrowing of int128_t. Thanks for catching it!